.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+# Overview
+
+Addresses ticket [#TICKET-NUMBER](www.ticket-link.com)
+
+High level overview of changes; one or two sentences.
+
+The design for the ticket is based on:
+![This design from XXXXXX](screenshot-of-design.png)
+
+# Changes
+
+## Major
+
+- Major change one
+- Major change two
+- ...
+
+## Minor
+
+- Minor change one
+- Minor change two
+- Minor change three
+- ...
+
+## Non-functional changes (e.g. documentation)
+
+- Docs change one
+
+# Release requirements
+
+What steps are required for release, if any? E.g. Key Vault secrets, Terraform changes, etc.
+
+## Release requirement one title
+
+Release requirement one description
+
+- Step one
+- Step two
+- ...
+
+# Additional notes
+
+Any additional notes, comments, etc. that might be required here.
+
+# Checklist
+
+- [ ] Title uses [Angular commit convention](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
+- [ ] PR targets development branch
+- [ ] Unit tests have been added/updated
+- [ ] E2E tests have been added/updated
+- [ ] GitHub workflows/actions have been updated
+- [ ] Terraform has been updated
+- [ ] Documentation has been updated
+- [ ] Key Vault secrets have been added to dev
```

Comment:
I'd err on the side of removing Github workflows/actions & terraform from this list as those feel more specific to a particular piece of work, wheras the others apply to all PRs but open to your thoughts on it

Reply:
I agree in principle, but it's easy for the PR creator to just remove the inappropriate lines right? 

How do you feel about adding something like: `Delete where appropriate` on the line before the checklist instead?

Reply:
Orrrr this:

```
## Checklist

Please delete checks below if not applicable.

- [ ] Title uses [Angular commit convention](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
- [ ] PR targets development branch
- [ ] Documentation has been updated where relevant

### Code 

- [ ] Unit tests have been added/updated
- [ ] E2E tests have been added/updated

### CI/CD

- [ ] GitHub workflows/actions have been added/updated
- [ ] Workflows/actions documentation has been added/updated

### IAC

- [ ] Terraform has been updated
- [ ] Terraform readme(s) have been updated
- [ ] Terraform documentation has been updated in Azure DevOps

### Infrastructure

- [ ] Any required Key Vault secrets have been added to the development Key Vault
```

Looks a lot then

Reply:
Yeah delete non applicable ones sounds good, that works 👍

Reply:
In principle its great but I think with everything else in the template the separate sections are too much, I'd keep them grouped like that but without the headers